juju/juju
# Add --force flag to remove-offers

## Description of change

This PR allows people to remove offers which have connections by adding a --force arg to remove-offers command.

## QA steps

Deploy a cross model scenario.
Remove an offer which has connections.
Ensure it fails without --force
With --force, ensure it works and also deletes the relations
Check relation departed hooks run etc

## Documentation changes

Need to document the new --force arg for remove-offers

## Bug reference

https://bugs.launchpad.net/juju/+bug/1757926
https://bugs.launchpad.net/juju/+bug/1757944
